README.md

```diff
@@ -1 +1,14 @@
-# Microverse-Works
\ No newline at end of file
+# Microverse-Works
+
+## Author 
+Solomon Khaluhi
+
+## Languages
+HTML
+CSS
+
+## About
+Integration of portflio cards using grids and flexbox
+
+## Contact
+khaluhi1997gmail.com
```

Comment:
your Readme file  is not looking  professional please try to use the [Template README ](https://github.com/microverseinc/readme-template) and customize it on your own 

also if you submit a PR without a pro README file in the future your PR will be invalid